optimum/bettertransformer/models/encoder_models.py

```diff
@@ -547,3 +547,211 @@ def forward(self, hidden_states, attention_mask, *_, **__):
         if hidden_states.is_nested and self.is_last_layer:
             hidden_states = hidden_states.to_padded_tensor(0.0)
         return (hidden_states,)
+
+
+class ViTLayerBetterTransformer(BetterTransformerBaseLayer):
+    r"""
```

Comment:
Should be moved under the `__init__`. Same issue for existing `WhisperEncoderLayerBetterTransformer`

Reply:
Fixed in  ad02275